src/v2i-hub/PedestrianPlugin/manifest.json

```diff
@@ -32,6 +32,29 @@
 			"key":"WebServicePort",
 			"default":"9000",
 			"description":"Port at which Web service exists"
+		},
+		{
+			"key":"DataProvider",
+			"default":"FLIR",
+			"description":"Possible data providers we handle - FLIR or PSM XML data"
```

Comment:
Is this a configuration to set whether we handle REST requests or WebSocket data from FLIR. If so configuration name and /or description can probably be improved to make this clear.

Reply:
Done